benefits/eligibility/templates/eligibility/start.html

```diff
@@ -6,37 +6,45 @@
 
 {% block main_content %}
   <div class="container">
-    <h1>{{ start_headline }}</h1>
+    <div class="row justify-content-center">
+      <div class="col-12 col-sm-12 col-lg-10">
+        <h1>{{ start_headline }}</h1>
 
-    <h2 class="media-title">{{ start_sub_headline }}</h2>
+        <h2 class="media-title p-sm">{{ start_sub_headline }}</h2>
 
-    {% block media_list %}
-      {% if media|length_is:"0" %}
-      {% else %}
-        <ul class="media-list">
-          {% for item in media %}
-            <li class="media">
-              <div class="media-line">{% include "core/includes/icon.html" with icon=item.icon %}</div>
-              <div class="media-body">
-                <h3 class="media-body--heading">{{ item.heading }}</h3>
-                <div class="media-body--details">
-                  <p>{{ item.details }}</p>
-                  <div class="media-body--items">
-                    <ul>
-                      {% for l in item.bullets %}<li>{{ l }}</li>{% endfor %}
-                    </ul>
+        {% block media_list %}
```

Comment:
I had the same thought as @thekaveman about replacing this with the include, but if you want to leave it alone for #1013 to take care of, that's fine with me